satellite/notification/service_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package notification_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
```

Comment:
`internal` -> `private`

Reply:
@stefanbenten that's the reason why gomobile build is failing on Jenkins

Reply:
Good catch!